app/views/errors/not_found.html.slim

```diff
@@ -0,0 +1,25 @@
+html[lang="en"]
+  head
+    meta[charset="utf-8"]
+    meta[http-equiv="X-UA-Compatible" content="IE=edge"]
+    meta[name="viewport" content="width=device-width, initial-scale=1"]
+    title
+      | The page you requested was not found
+    link[href="https://fonts.googleapis.com/css?family=Cabin:400,700" rel="stylesheet"]
+    link[href="https://fonts.googleapis.com/css?family=Montserrat:900" rel="stylesheet"]
+    link[type="text/css" rel="stylesheet" href="css/style.css"]
```

Comment:
maybe move header to _header.html.slim and reder it for each errors files

Reply:
Don't render `head` in views
`head` must render only in layout

Reply:
If you need some fonts on specific pages you can import it in appropriate .scss files